.template/addons/docker/Dockerfile.tt

```diff
@@ -31,7 +31,7 @@ ENV LANG="en_US.UTF-8" \
     LANGUAGE="en_US:en"
 
 RUN apt-get update -qq && \
-    apt-get install -y --no-install-recommends apt-transport-https curl gnupg
+    apt-get install -y --no-install-recommends apt-transport-https curl gnupg net-tools
```

Comment:
Need to install this package to use `netstat` which is used by the serverspec to check if the port is listening - https://github.com/mizzy/specinfra/blob/8a19cf4840d2240e738c7f300e29fbe373f5fda7/spec/command/debian/port_spec.rb